src/ethereum/osaka/vm/instructions/environment.py

```diff
@@ -353,6 +357,10 @@ def extcodesize(evm: Evm) -> None:
 
     # OPERATION
     code = get_account(evm.message.block_env.state, address).code
+    
+    # BAL tracking for address access
```

Comment:
These comments don't add any information that isn't obvious from the code below them.